--- ./SConstruct.orig
+++ ./SConstruct
@@ -5003,7 +5003,8 @@
     if (conf.CheckCXXHeader("execinfo.h")
             and conf.CheckDeclaration('backtrace', includes='#include <execinfo.h>')
             and conf.CheckDeclaration('backtrace_symbols', includes='#include <execinfo.h>')
-            and conf.CheckDeclaration('backtrace_symbols_fd', includes='#include <execinfo.h>')):
+            and conf.CheckDeclaration('backtrace_symbols_fd', includes='#include <execinfo.h>')
+            and conf.CheckFunc('backtrace')):
 
         conf.env.SetConfigHeaderDefine("MONGO_CONFIG_HAVE_EXECINFO_BACKTRACE")
 
